src/main/java/frc/robot/subsystems/AlgeaRemover.java

```diff
@@ -0,0 +1,153 @@
+import static edu.wpi.first.units.Units.RadiansPerSecond;
+import static edu.wpi.first.units.Units.Rotations;
+
+import com.revrobotics.sim.SparkMaxSim;
+import com.revrobotics.spark.SparkBase.ControlType;
+import com.revrobotics.spark.SparkBase.PersistMode;
+import com.revrobotics.spark.SparkBase.ResetMode;
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+import com.revrobotics.spark.SparkMax;
+import com.revrobotics.spark.config.ClosedLoopConfig;
+import com.revrobotics.spark.config.ClosedLoopConfig.FeedbackSensor;
+import com.revrobotics.spark.config.SparkMaxConfig;
+import edu.wpi.first.units.measure.Angle;
+import edu.wpi.first.wpilibj.RobotController;
+import edu.wpi.first.wpilibj.shuffleboard.ComplexWidget;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj.util.Color8Bit;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Constants.AlgeaRemoverConstants;
+import frc.robot.Robot;
+
+public class AlgeaRemover extends SubsystemBase {
+  private SingleJointedArmSim algeaSim;
+  private SparkMax algeaMotor;
+  private SparkMaxSim simAlgeaMotor;
+  private Angle simAngle;
+
+  private Mechanism2d mech = new Mechanism2d(2, 2);
+  private ComplexWidget widget;
+  private MechanismLigament2d algeaMech;
+
+  public static final ClosedLoopConfig algeaCfg =
+      new ClosedLoopConfig()
+          .feedbackSensor(FeedbackSensor.kPrimaryEncoder)
+          .pid(
+              AlgeaRemoverConstants.kAlgeaP,
+              AlgeaRemoverConstants.kAlgeaI,
+              AlgeaRemoverConstants.kAlgeaD);
+
+  public AlgeaRemover() {
+    algeaMotor = new SparkMax(Constants.CANIDs.kAlgeaMotor, MotorType.kBrushless);
+    algeaMotor.configure(
+        new SparkMaxConfig().apply(algeaCfg),
+        ResetMode.kNoResetSafeParameters,
+        PersistMode.kNoPersistParameters);
+    if (Robot.isSimulation()) {
+      simAlgeaMotor = new SparkMaxSim(algeaMotor, AlgeaRemoverConstants.kAlgeaGearbox);
+      algeaSim =
+          new SingleJointedArmSim(
+              AlgeaRemoverConstants.kAlgeaGearbox,
+              AlgeaRemoverConstants.kAlegeaGearRatio,
+              SingleJointedArmSim.estimateMOI(
+                  AlgeaRemoverConstants.algeaArmLength.in(Meters), .5), // Update with real value
+              AlgeaRemoverConstants.algeaArmLength.in(Meters),
+              -Math.PI / 2,
+              Math.PI / 4,
+              true,
+              -Math.PI / 2);
+      simAlgeaMotor.setPosition(-.25 * AlgeaRemoverConstants.kAlegeaGearRatio);
```

Comment:
Make an Angle constant for your starting position and use it for the SingleJointedArmSim bounds and setPosition